src/Header/SetCookie.php

```diff
@@ -572,6 +612,30 @@ public function isSessionCookie()
         return ($this->expires === null);
     }
 
+    /**
+     * @return string|null
+     */
+    public function getSameSite()
+    {
+        return $this->sameSite;
+    }
+
+    /**
+     * @param string|null $sameSite
+     * @return SetCookie
+     * @throws Exception\InvalidArgumentException
+     */
+    public function setSameSite($sameSite)
+    {
+        if (! in_array($sameSite, self::SAME_SITE_ALLOWED_VALUES, true)) {
+            throw new Exception\InvalidArgumentException(
+                '"' . $sameSite . '" is not an allowed value (Strict, Lax, None) for SameSite directive.'
+            );
```

Comment:
Need unit test for that case, and we are good :) thx

Reply:
ok :)

Reply:
Done :-)